include/boost/geometry/algorithms/discrete_frechet_distance.hpp

```diff
@@ -242,7 +242,8 @@ struct discrete_frechet_distance<default_strategy, false>
 [heading Available Strategies]
 \* [link geometry.reference.strategies.strategy_distance_pythagoras Pythagoras (cartesian)]
 \* [link geometry.reference.strategies.strategy_distance_haversine Haversine (spherical)]
-[/ \* more (currently extensions): Vincenty\, Andoyer (geographic) ]
+\* One of the geographic point to point strategies
```

Comment:
We still should document `Vincenty` and others

Reply:
Should we add a TODO here (although it is more general task than similarity distances)

Reply:
I'll make a separate issue for it.